app/views/comments/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h2>Add a Comment</h2>
+
+<%= image_tag "http://static.deathandtaxesmag.com/uploads/2014/02/Screen-Shot-2014-02-19-at-8.33.46-PM-585x377.png", alt: "sinatra_pic", size: "600 x 500" %>
+<%= render partial: "form" %>
```

Comment:
nice use of partial! you can exclude the word here if you like: `<%= render "form" %>`